### PR TITLE
Feature user repository

### PR DESCRIPTION
**Pull request by JoelIvanStella**
## Summary
- Adding user repository
## Trello Card
https://trello.com/c/28iSxF1y
## Test
image

### CODE REVIEW

**Review by torsello — APPROVED** [review]
LGTM! :pick: :pick:

File: src/main/java/com/wolox/training/repositories/UsersRepository.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.wolox.training.models.Users;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UsersRepository extends JpaRepository<Users, Long> {
+
+    public Optional<Users> findByUserName(String userName);
```
Comment (jdhiguita) [inline src/main/java/com/wolox/training/repositories/UsersRepository.java]: It's no necessary add public to method in the repositories.

**Review by jdhiguita — COMMENTED** [review]
Great job! One single comment

File: src/main/java/com/wolox/training/repositories/UsersRepository.java (hunk 1 above)
Comment (JoelIvanStella) [reply src/main/java/com/wolox/training/repositories/UsersRepository.java]: Yes, thanks! I pushed the change

**Review by jdhiguita — APPROVED** [review]

**Pull request merged**